course/templates/course/module.html

```diff
@@ -24,13 +24,11 @@
         <br />
         <span class="dates">
             <em>
-                {% blocktrans with deadline=current.late_time|date:"DATETIME_FORMAT" %}
-                Late submissions are allowed until {{ deadline }}.
-                {% endblocktrans %}
+                {% blocktranslate trimmed with deadline=current.late_time|date:"DATETIME_FORMAT" %}
+                    LATE_SUBMISSIONS_ALLOWED_UNTIL -- {{ deadline }}{% endblocktranslate %}
```

Comment:
These {% endblocktranslate %} tags should probably be on their own rows, since the {% blocktranslate %} tags are on their own rows.

Reply:
Good catch, thanks, I'll fix these!